src/models/UserModel.ts

```diff
@@ -57,39 +66,53 @@ export class UserModel {
     name: "user_blocking_users",
     joinColumn: {
       name: "blockers",
-      referencedColumnName: "id"
+      referencedColumnName: "id",
     },
     inverseJoinColumn: {
       name: "blocking",
-      referencedColumnName: "id"
-    }
+      referencedColumnName: "id",
+    },
   })
   blocking: UserModel[] | undefined;
 
   @ManyToMany(() => UserModel, (user) => user.blocking)
   blockers: UserModel[] | undefined;
 
-  @OneToMany(() => PostModel, post => post.user, { cascade: true })
+  @OneToMany(() => ReportModel, (report) => report.reporter)
+  reports: ReportModel[];
+
+  @OneToMany(() => ReportModel, (report) => report.reported)
+  reportedBy: ReportModel[];
+
+  @OneToMany(() => PostModel, (post) => post.user, { cascade: true })
   posts: PostModel[];
 
-  @ManyToMany(() => PostModel, post => post.savers)
+  @ManyToMany(() => PostModel, (post) => post.savers)
   saved: PostModel[];
 
-  @OneToMany(() => UserSessionModel, session => session.user, { cascade: true })
+  @OneToMany(() => UserSessionModel, (session) => session.user, {
+    cascade: true,
+  })
   sessions: UserSessionModel[];
 
-  @OneToMany(() => FeedbackModel, feedback => feedback.user)
+  @OneToMany(() => FeedbackModel, (feedback) => feedback.user)
   feedbacks: FeedbackModel[];
 
-  @OneToMany(() => RequestModel, request => request.user)
+  @OneToMany(() => RequestModel, (request) => request.user)
   requests: RequestModel[];
 
-  @OneToMany(() => UserReviewModel, review => review.buyer)
+  @OneToMany(() => UserReviewModel, (review) => review.buyer)
   reviewsWritten: UserReviewModel[];
 
-  @OneToMany(() => UserReviewModel, review => review.seller)
+  @OneToMany(() => UserReviewModel, (review) => review.seller)
   reviewsReceived: UserReviewModel[];
 
+  // @OneToMany(() => MessageModel, (message) => message.sender)
+  // sentMessages: MessageModel[];
+
+  // @OneToMany(() => MessageModel, (message) => message.receiver)
+  // receivedMessages: MessageModel[];
+
```

Comment:
Same comment here about deleting commented code

Reply:
yeah gonna be using this in the future when i build the messages model